views/urls_test.go

```diff
@@ -0,0 +1,79 @@
+package views
```

Comment:
Added a go test file for this. Is this intended to be run manually only or are these run in a pipeline as well?

Reply:
manual for now, but would like to change that